misk/src/main/kotlin/misk/moshi/wire/MapAdapter.kt

```diff
@@ -0,0 +1,18 @@
+package misk.moshi.wire
+
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.JsonReader
+import com.squareup.moshi.JsonWriter
+
+internal class MapAdapter(
+        private val keyAdapter: JsonAdapter<Any>,
+        private val valueAdapter: JsonAdapter<Any>
+) : JsonAdapter<Map<Any, Any>>() {
+    override fun fromJson(reader: JsonReader): Map<Any, Any>? {
+        throw UnsupportedOperationException()
```

Comment:
Todo?

Reply:
Removed - we don't actually need this, can rely on moshi to do the right thing with maps. Added test covering maps